Objects/setobject.c

```diff
@@ -488,8 +516,9 @@ static void
 set_dealloc(PySetObject *so)
 {
     setentry *entry;
-    Py_ssize_t used = so->used;
+    Py_ssize_t used;
 
+    used = so->used;
```

Comment:
Revert unnecessary change:

```suggestion
    Py_ssize_t used = so->used;
```